Wellthier/View Controllers/HealthKitWorkoutsViewController.m

```diff
@@ -38,6 +38,11 @@ - (void)getWorkouts {
     }];
 }
 
+- (IBAction)didTapDone:(id)sender {
+    [self dismissViewControllerAnimated:true completion:nil];
+    [self.delegate didPickWorkout:self.currentlySelectedCell];
```

Comment:
We should not pass UI objects, can we parse the information needed to appear in the screen and only pass it through delegate

Reply:
May be HKWorkout object should be enough to pass down ?